partners.yml

```diff
@@ -0,0 +1,546 @@
+          type: number
+          description: Repayment time requested in months
+          minimum: 0
+          exclusiveMinimum: true
+          example: 24
+        reasonForLoan:
+          type: string
+          description: Purpose of the loan. The enum values will be provided to the partner.
+          enum:
+          - vehicle
+          - refinancing
+          - rennovation
+          - ...
+        isBorrowingForOwnUse:
+          type: boolean
+          description: Indicate if customer is asking the loan for himself
+          example: true
+        applicants:
+          type: array
+          items:
+            $ref: "#/components/schemas/applicant"
+
+    extendedApplication:
+      allOf:
+        - $ref: "#/components/schemas/application"
+        - type: object
+          required:
+          - status
+          - decision
+          - loanTerms
+          properties:
+            status:
+              type: string
+              description: Status of the application. The enum values will be provided to the partner.
+              enum:
+              - pending
+              - payout
+              - ...
+            decision:
+              type: string
+              description: Decision taken on the application. The enum values will be provided to the partner.
+              enum:
+              - deny
+              - preliminary-granted
+              - granted
+              - ...
+            loanTerms:
+              $ref: "#/components/schemas/loanTerms"
+
+    applicant:
+      type: object
+      required:
+      - socialSecurityNumber
+      - email
+      - phone
+      - isNotPep
+      - economicDetails
+      properties:
+        socialSecurityNumber:
+          type: string
+          format: YYYYMMDDXXXX
+          description: Swedish personnummer in format YYYYMMDDXXXX
+          example: "197202104332"
+        email:
+          type: string
+          description: Email of the applicant. Used to communicate loan agreement
+          pattern: /^(([^<>()[\].,;:\s@"]+(\.[^<>()[\].,;:\s@"]+)*)|(".+"))@(([^<>()[\].,;:\s@"]+\.)+[^<>()[\].,;:\s@"]{2,})$/
+          example: test@example.com
+        phone:
+          type: string
+          description: Phone number of the applicant (preferably mobile phone). Used to communicate loan agreement if it is a mobile phone number
+          format: international or swedish format
+          example: "+46731234567"
+        isNotPep:
+          type: boolean
+          description: Indicate if the customer is politically exposed
+          example: true
+        economicDetails:
+          $ref: "#/components/schemas/economicDetails"
+        bankDetails:
+          $ref: "#/components/schemas/bankDetails"
+        employer:
+          $ref: "#/components/schemas/employer"
+    
+    economicDetails:
+      type: object
+      required:
+      - monthlyIncome
+      - monthlyHousing
+      - monthlyLoans
+      - occupation
+      - kids
```

Comment:
Maybe add the `homeType` as required as well. We have it in our frontend so why not require it from partner for now.